### PR TITLE
COP-2493: Implement the new design for voyage people

### PR DESCRIPTION
**Pull request by rafenden**
## Description

Implement the new design for selecting voyage passengers which involves adding one more step to the voyage form.

Depends on https://github.com/UKHomeOffice/sgmr-service/pull/181

## Testing

1. Install dependencies `npm i`;
2. Start creating new voyage report;
3. Go to the step no 4 (voyage people);
4. Test adding a new person and removing exiting passangers;

## Developer Checklist

\* Required

- [x] Up-to-date with master branch? \*

- [x] Does it have tests?

- [x] Pull request URL linked to JIRA ticket? \*

- [ ] All reviewer comments resolved/answered? \*

Ensure you delete the branch once the Pull Request is merged.

### CODE REVIEW

**Review by ajames14 — COMMENTED** [review]
All seems to be working and code looks good to me but the cypress tests appear to be failing (the add-people and add-voyage tests).

**Comment by rafenden** [comment]
> All seems to be working and code looks good to me but the cypress tests appear to be failing (the add-people and add-voyage tests).

Fixed now :)

**Review by ajames14 — DISMISSED** [review]

File: src/components/User/UserRegister.jsx [hunk 1]
```diff
@@ -117,7 +117,7 @@ const UserRegister = () => {
                   Mobile number
                 </label>
                 <FormError error={errors.mobileNumber} />
-                <span className="govuk-hint">We will send an access code to this number. For international numbers include the country code.</span>
+                <span className="govuk-hint">We will send an access code to this number</span>
                 <input
```
Comment (ajames14) [inline src/components/User/UserRegister.jsx:121]: It looks like this is undoing the changes I just merged to master?

File: src/components/User/UserRegisterValidation.jsx [hunk 2]
```diff
@@ -23,8 +22,8 @@ const UserRegisterValidation = (dataToValidate) => {
   }
 
   // Mobile Number field must be valid
-  if (!(VALID_MOBILE_REGEX.test(dataToValidate.mobileNumber))) {
-    fieldsErroring.mobileNumber = 'You must enter a valid phone number e.g. 07700 900982, +33 63998 010101';
+  if (!parseInt(dataToValidate.mobileNumber, 10)) {
```
Comment (ajames14) [inline src/components/User/UserRegisterValidation.jsx:25]: And here

File: src/components/User/UserRegister.jsx (hunk 1 above)
Comment (rafenden) [reply src/components/User/UserRegister.jsx:121]: You are right, let me rebase again.

**Review by ajames14 — APPROVED** [review]

**Pull request merged**